ignore.js

```diff
@@ -52,7 +52,7 @@ const getIsIgnoredPredicate = (files, cwd) => {
 	return fileOrDirectory => {
 		fileOrDirectory = toPath(fileOrDirectory);
 		fileOrDirectory = toRelativePath(fileOrDirectory, cwd);
-		return ignores.ignores(slash(fileOrDirectory));
+		return fileOrDirectory && ignores.ignores(slash(fileOrDirectory));
```

Comment:
Let's wrap the result with `Boolean` so it can match the type definition?

Reply:
@fisker Fixed + I also fixed the tests that were failing on windows